bitnami/redis/README.md

```diff
@@ -337,7 +337,8 @@ The command removes all the Kubernetes components associated with the chart and
 | `pdb.maxUnavailable`                    | Max number of pods that can be unavailable after the eviction                                                       | `nil`   |
 | `tls.enabled`                           | Enable TLS traffic                                                                                                  | `false` |
 | `tls.authClients`                       | Require clients to authenticate                                                                                     | `true`  |
-| `tls.certificatesSecret`                | Then name of the existing secret that contains the TLS certificates                                                 | `nil`   |
+| `tls.autoGenerated`                     | Generate automatically self-signed TLS certificates                                                                 | `nil`   |
+| `tls.existingSecret`                    | The name of the existing secret that contains the TLS certificates                                                  | `nil`   |
```

Comment:
I think some kind of testing should be added to check whether both are set at the same time or not.
Also, for `autogenerated`, it should be indicated that it is a boolean value.

Reply:
I have modified the helper "redis.createTlsSecret".
Although `autoGenerated=true`, it will be ignored if an `existingSecret`or `certificatesSecret`  is provided.

```
{{/*
Return true if a TLS secret object should be created
*/}}
{{- define "redis.createTlsSecret" -}}
{{- if and .Values.tls.enabled .Values.tls.autoGenerated (and (not .Values.tls.existingSecret) (not .Values.tls.certificatesSecret)) }}
    {{- true -}}
{{- end -}}
{{- end -}}
```

Reply:
> I think some kind of testing should be added to check whether both are set at the same time or not.

Additionally, I have added a validation check to trigger an error when `tls.enabled=true`, `tls.autogenerated=false`, and no existing secret is provided.

Reply:
Testing output
```console
$helm install redis bitnami/redis --set tls.enabled=true,auth.tls.autoGenerated=false,tls.certCAFilename=/this/test,tls.authClients=false,tls.certFilename=/this/test,tls.certKeyFilename=/this/tes
Error: template: redis/templates/NOTES.txt:147:4: executing "redis/templates/NOTES.txt" at <include "redis.validateValues" .>: error calling include: template: redis/templates/_helpers.tpl:234:51: executing "redis.validateValues" at <fail>: error calling fail: 
VALUES VALIDATION:

redis: tls.enabled
    In order to enable TLS, you also need to provide
    an existing secret containing the TLS certificates or
    enable auto-generated certificates.
```

Reply:
You should indicate in the README the default vault for `tls.autoGenerated`